packages/api/src/routes/medical/network.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import {
+  getHieDirectoryEntriesByFilter,
+  getHieDirectoryEntriesByFilterCount,
+} from "../../external/carequality/command/cq-directory/get-cq-directory-entries";
+import { requestLogger } from "../helpers/request-logger";
+import { asyncHandler } from "../util";
+import { dtoFromHieDirectoryEntry, NetworkDTO } from "./dtos/networkDTO";
+import { networkGetSchema } from "./schemas/network";
+
+import { PaginatedResponse } from "@metriport/shared";
+import { Pagination } from "../../command/pagination";
+import { paginated } from "../pagination";
+
+const router = Router();
+
+/** ---------------------------------------------------------------------------
+ * GET /network
+ *
+ * Gets all HIE directory entries (AKA Networks) corresponding that Metriport has access to.
+ *
+ * @param   req.cxId              The customer ID.
+ * @param   req.query.filter      Full text search filters. See https://docs.metriport.com/medical-api/more-info/search-filters
+ * @param   req.query.fromItem    The minimum item to be included in the response, inclusive.
+ * @param   req.query.toItem      The maximum item to be included in the response, inclusive.
+ * @param   req.query.count       The number of items to be included in the response.
+ * @return  The customer's HIE directory entries (AKA Networks).
+ */
+router.get(
+  "/",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const params = networkGetSchema.parse(req.query);
+    const additionalQueryParams = params.filter ? { filter: params.filter } : undefined;
+
+    const { meta, items } = await paginated({
+      request: req,
+      additionalQueryParams,
+      getItems: (pagination: Pagination) => {
+        return getHieDirectoryEntriesByFilter({ filter: params.filter || "", pagination });
```

Comment:
Let's not use empty string as fallback, `undefined` represents this better - [context](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#cff793e9f7f5445fb075c82490040bc1).

Reply:
Thank you for the lesson oh master 🙇 this is something I've been guilty of for years! helped me understand _why_ its bad